cocos2d/videoplayer/video-player-impl.js

```diff
@@ -158,23 +158,24 @@ let VideoPlayerImpl = cc.Class({
         video.style.height = height + 'px';
     },
 
-    _createDom () {
+    _createDom (mute, volume) {
         let video = document.createElement('video');
         video.style.position = "absolute";
         video.style.bottom = "0px";
         video.style.left = "0px";
         video.className = "cocosVideo";
         video.setAttribute('preload', 'auto');
         video.setAttribute('webkit-playsinline', '');
-        // This x5-playsinline tag must be added, otherwise the play, pause events will only fire once, in the qq browser.
-        video.setAttribute("x5-playsinline", '');
         video.setAttribute('playsinline', '');
+        if (mute || volume === 0) {
+            video.setAttribute('muted', '');
+        }
 
         this._video = video;
         cc.game.container.appendChild(video);
     },
 
-    createDomElementIfNeeded: function () {
+    createDomElementIfNeeded: function (mute, volume) {
         if (!this._video) {
             this._createDom();
```

Comment:
You forgot to pass the argument to _createDom

Reply:
Added here as well 👍